### DISCUSSION TITLE
new-admin not working

### DISCUSSION THREAD

**Discussion by 12nick12** [modoboa/modoboa#2574] [Help]
After updating `The requested page does not exist.` when going to new-admin

**Comment by tonioo** [comment]
Have you updated your nginx configuration?

**Comment by LefflerMediaLLC** [comment]
No I didn't. Do you have a copy of what the new config needs added?

**Comment by 12nick12** [comment]
Does there happen to be a copy of what needs to be updated?

**Comment by Spitfireap** [comment]
You can take a look here https://github.com/modoboa/modoboa-installer/blob/b391de5937e98571f03a6f6388d8ec8a3fe8ff87/modoboa_installer/scripts/files/nginx/modoboa.conf.tpl#L40 :)

**Comment by 12nick12** [comment]
Thank you, where it happen to be documented anywhere how I can get the frontend folder? It appears it wasn't installed when I upgraded?

**Reply by tonioo** [reply]
https://modoboa.readthedocs.io/en/latest/upgrade.html#new-admin-interface

**Reply by bernd-wechner** [reply]
Faced this same problem myself. As I don't use nginx (nor want to) and so read that template, and it's a one-liner in lighttpd as far as I can tell:

`alias.url = ( "/new-admin" => "<the document root>/frontend" )`

And the v2 interface now works (though it's a mild shame I find to see a Django project move outside of the Django URL resolver).

But to be clear (as I'm not a fan of presumptive webserver configs and prefer explanation of intended purpose) what is needed here is to configure your webserver to mapp all of the `/new-admin` URLS to the `/frontend` folder in the Django web root and NOT pass them to the WSGI service (and the Django URL handler). How you do that will depend on the webserver.

**Comment by cdaraujo1** [comment]
this is not working with docker. even the `/frontend` doesn't work. does someone have a solution?
